span-normalizer/span-normalizer/src/test/resources/configs/span-normalizer/application.conf

```diff
@@ -51,3 +52,8 @@ processor {
   ]
 }
 
+processor {
+  bypass.key = "test.bypass"
```

Comment:
Any suggestions on naming this?

Reply:
this refers to the span attribute ?

Reply:
Yes. And that attribute's true value will be considered for bypassing the traffic.
So, in this example, the test is checking if span attribute `test.bypass` exists and its value is `true`, it will directly give traffic to the enricher.

Reply:
Do we need to consider process tags too?

Reply:
why the processor construct here ?